### DISCUSSION TITLE
Chrome dialog

### DISCUSSION THREAD

**Discussion by jonathangiber** [getgauge/taiko#2634] [General]
Hey guys,
Is there any way to handle a chrome dialog with taiko? 

Example attached 
![VlJTV](https://user-images.githubusercontent.com/7656511/185129488-82915169-aaf0-434d-a6d7-d61e83f0d17b.png)

**Comment by NivedhaSenthil** [comment]
you can use the encrypted password when navigating through `goto` . Please refer https://spectrum.chat/taiko/general/how-to-load-a-site-with-basic-authorization~77ec9623-3964-452b-a027-550bbe7b0c69

**Comment by jonathangiber** [comment]
Hi @NivedhaSenthil,
unfortunately this isn't good enough for me since my application doesnt do Basic authentication, it does Digest authentication, thus those headers arent good enough.

So there is no way to handle it?

**Comment by jonathangiber** [comment]
Could this work https://docs.taiko.dev/api/switchto/? or its not supported?

**Reply by NivedhaSenthil** [reply]
`SwitchTo` would not solve this use case, let me check and find a resolution for this
